exist-ant/src/main/java/org/exist/ant/RestoreTask.java

```diff
@@ -47,7 +47,7 @@
     private Path   dir             = null;
     private DirSet dirSet          = null;
     private String restorePassword = null;
-    private boolean overwriteApps = false;
+    private final boolean overwriteApps = false;
```

Comment:
interesting

Reply:
jep, guess this change was made by the IDE though. Still interesting though...